src/runtime/ze/ze_hardware_manager.cpp

```diff
@@ -263,8 +263,8 @@ bool ze_hardware_context::has(device_support_aspect aspect) const {
   case device_support_aspect::global_mem_cache_read_only:
     return false;
     break;
-  case device_support_aspect::global_mem_cache_write_only:
-    return false;
+  case device_support_aspect::global_mem_cache_read_write:
+    return true;
```

Comment:
I couldn't actually find a reference stating that all devices that can be used with Level Zero have read/write cache but dpc++ returns true here, and I think we can trust that this is true then.

Reply:
Agreed :)